DuckDuckGo/HomeViewController+DaxDialogs.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  HomeViewController+DaxDialogs.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import UIKit
+import SwiftUI
+
+extension HomeViewController {
+
+    func showNextDaxDialog(dialogProvider: NewTabDialogSpecProvider) {
+        guard let spec = dialogProvider.nextHomeScreenMessage() else { return }
+        guard !isShowingDax else { return }
+        guard let daxDialogViewController = daxDialogViewController else { return }
+        collectionView.isHidden = true
+        daxDialogContainer.isHidden = false
+        daxDialogContainer.alpha = 0.0
+
+        daxDialogViewController.loadViewIfNeeded()
+        daxDialogViewController.message = spec.message
+        daxDialogViewController.accessibleMessage = spec.accessibilityLabel
+
+        view.addGestureRecognizer(daxDialogViewController.tapToCompleteGestureRecognizer)
+
+        daxDialogContainerHeight.constant = daxDialogViewController.calculateHeight()
+        hideLogo()
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```

Comment:
Where did this delay come from?  Was it always there?  It was surprising to me when I saw it, I would consider just making it smaller (less than 0.3 which is the threshold for being noticeable usually) if it's really needed or just get rid of it entirely.

Reply:
This is the old onboarding, I just moved the method but I have not touched anything about it.